infra-as-code/bicep/acr.bicep

```diff
@@ -13,16 +14,41 @@ param location string = resourceGroup().location
 @description('The name of the virtual network that this ACR instance will have a private endpoint in.')
 param vnetName string
 
-@description('The name of the subnet for the private endpoint. Must in in the provided virtual network.')
+@description('The name of the subnet for the private endpoint. Must be in the provided virtual network.')
 param privateEndpointsSubnetName string
 
+@description('The name of the build node agent.')
+param acrBuildAgentNodeName string
+
+@description('The list of tiers for the Linux node agent.')
+param linuxNodeAgentTier string
+
 @description('The name of the workload\'s existing Log Analytics workspace.')
 param logWorkspaceName string
 
 // Variables
 var acrName = 'cr${baseName}'
+var acrManagedIdentityName = 'id-${acrName}'
 var acrPrivateEndpointName = 'pep-${acrName}'
 var acrDnsZoneName = 'privatelink${environment().suffixes.acrLoginServer}'
+var acrSubnetID = resourceId('Microsoft.Network/VirtualNetworks/subnets', vnetName, privateEndpointsSubnetName)

```

Comment:
Please move this to be an `existing` resource, like you see on line `57`

Reply:
Also, this isn't the correct subnet.

Also, this subnet needs firewall rules adjusted per <https://learn.microsoft.com/azure/container-registry/tasks-agent-pools#create-pool-in-a-virtual-network>